[2024-05-24T12:00:24.660+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-24T12:00:24.669+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2023-06-18T00:00:00+00:00 [queued]>
[2024-05-24T12:00:24.673+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2023-06-18T00:00:00+00:00 [queued]>
[2024-05-24T12:00:24.673+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-24T12:00:24.678+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_and_upload_to_s3> on 2023-06-18 00:00:00+00:00
[2024-05-24T12:00:24.681+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=436) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-24T12:00:24.681+0000] {standard_task_runner.py:63} INFO - Started process 442 to run task
[2024-05-24T12:00:24.682+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'upload_from_github_to_s3', 'download_and_upload_to_s3', 'scheduled__2023-06-18T00:00:00+00:00', '--job-id', '367', '--raw', '--subdir', 'DAGS_FOLDER/upload_from_github_to_s3_dag.py', '--cfg-path', '/tmp/tmp4d9k4r_e']
[2024-05-24T12:00:24.682+0000] {standard_task_runner.py:91} INFO - Job 367: Subtask download_and_upload_to_s3
[2024-05-24T12:00:24.702+0000] {task_command.py:426} INFO - Running <TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2023-06-18T00:00:00+00:00 [running]> on host b2ab95bd73fd
[2024-05-24T12:00:24.739+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_from_github_to_s3' AIRFLOW_CTX_TASK_ID='download_and_upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2023-06-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-18T00:00:00+00:00'
[2024-05-24T12:00:24.741+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-24T12:00:25.329+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-05-24T12:00:25.385+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-24T12:00:25.386+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-24T12:00:25.390+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=upload_from_github_to_s3, task_id=download_and_upload_to_s3, run_id=scheduled__2023-06-18T00:00:00+00:00, execution_date=20230618T000000, start_date=20240524T120024, end_date=20240524T120025
[2024-05-24T12:00:25.433+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-24T12:00:25.440+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-24T12:00:25.441+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
